Core/src/main/java/com/plotsquared/core/util/PlayerManager.java

```diff
@@ -173,7 +177,9 @@ public static void getUUIDsFromString(
      * @param owner    Owner UUID
      * @param blocking Whether or not the operation can be blocking
      * @return The player's name, None, Everyone or Unknown
+     * @deprecated Use {@link #resolveName(UUID, boolean)}
      */
+    @Deprecated(forRemoval = true, since = "TODO")
```

Comment:
Is there something wrong with the TODO here?

Reply:
It's a post-merge thing https://github.com/IntellectualSites/PlotSquared/pull/3449#discussion_r783457457

Reply:
> Can/should it not be replaced with the version? Or is that a post-merge thing

Yeah, I replace `@since TODO` with the version released. Using the current version can lead to conflicts or doesn't display the appropriate version, e.g. if this TODO is 1.2.1 but another PR merged would bump to 1.3.0, we would have to address the 1.2.1 ones post merge, do it on the other PR or let the PR author do it; generally introduces more work and more to take core of, instead of simply replacing TODOs when doing releases.